### ISSUE TITLE
More specific selectors 

### ISSUE THREAD

**Issue by orsinium** [facebookincubator/Bowler#94]
Now selectors captures everything related to the given topic. For example, `select_module` captures `module_name`, `module_nickname` etc. Sometimes it's useful to be more specific. For example:

```python
...select_module('test', capture=['module_name'])
```

**Comment by thatch** [comment]
Are you just suggesting prune the captures dict?  Could you give an example of where this is helpful?

**Comment by orsinium** [comment]
Yes. For example, when I want to get rid of "as" import. I can write my own selector, but it would be great to be able to reuse existent selectors from Bowler when possible.
